sdcm/cluster_gce.py

```diff
@@ -123,8 +124,15 @@ def reboot(self, hard=True, verify_ssh=True):
         pre_uptime = result.stdout
 
         def uptime_changed():
-            result = self.remoter.run('uptime -s', ignore_status=True)
-            return pre_uptime != result.stdout
+            try:
```

Comment:
why do we have the same code in two places COPY-PASTED? move this to the base node!

Reply:
Added this patch (https://github.com/scylladb/scylla-cluster-tests/pull/1081#issuecomment-506538835) to the PR.